docker-app/qfieldcloud/core/models.py

```diff
@@ -987,6 +993,36 @@ def delete_project(sender: Type[Project], instance: Project, **kwargs: Any) -> N
         qfieldcloud.core.utils2.storage.remove_project_thumbail(instance)
 
 
+class ProjectCollaboratorQueryset(models.QuerySet):
+    def validate(self, keep=False):
```

Comment:
```suggestion
    def validated(self, keep_invalid=False):
```

I would choose a different name, what if you go for "validated"? When you say "validate", I would expect to either return `true`/`false`, or to throw an Exception.